jupyterlab_search_replace/search_engine.py

```diff
@@ -22,8 +22,12 @@
 MAX_LOG_OUTPUT = 6000  # type: int
 
 
-def construct_command(query, max_count, case_sensitive, whole_word, include, exclude):
-    command = ["rg", "-e", query, "--json", f"--max-count={max_count}"]
+def construct_command(
+    query, max_count, case_sensitive, whole_word, include, exclude, use_regex
+):
+    command = ["rg", query, "--json", f"--max-count={max_count}"]
+    if not use_regex:
+        command.insert(1, "-F")
```

Comment:
As we tend to not use regex by default, I would do the opposite:

```suggestion
    command = ["rg", "-F", query, "--json", f"--max-count={max_count}"]
    if use_regex:
        command.remove("-F")
```